### ISSUE TITLE
It's impossible to change the scroll position when starting the application.

### ISSUE THREAD

**Issue by awp-sirius** [dotnet/maui#20611]
### Description

I have a task to automatically change the position of the scroll on the first page after launching the application.
I'm using the "OnAppearing" method override, but it's not working for my task.
I assume that this is due to the fact that at the moment when "OnAppearing" is triggered, the SplashScreen is still shown.

The problem occurs only once when the application is launched.
When navigating from other pages, the code works correctly.

### Steps to Reproduce

1. Create a new Maui application
2. Add content to the VerticalStackLayout in MainPage.xaml so that the elements do not fit on one screen.
3. Place VerticalStackLayout inside ScrollView with x:Name="MainScroll"
4. In MainPage.xaml.cs, override the OnAppearing method:
```C#
protected override async void OnAppearing()
{
    base.OnAppearing();
    await MainScroll.ScrollToAsync(0, 50, false);
}
```

Waiting: displaying a page with a scroll offset of 50.
Reality: the scroll position has not been changed.

P.S. I tried using "MainThread.BeginInvokeOnMainThread" but it didn't change the result.

### Version with bug

8.0.7 SR2

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 14.0

**Comment by PureWeen** [comment]
@awp-sirius does this work if you use onloaded vs onappearing?

**Comment by awp-sirius** [comment]
@PureWeen Yes, I tried all the events and they all trigger before the first page is shown.
